src/command_modules/azure-cli-hdinsight/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+0.3.5
++++++
+* Support --ssh-public-key parameter. Minor fixes.
```

Comment:
You can remove the "Minor fixes" part. You should be more specific about which commands support `--ssh-public-key`

Reply:
> You can remove the "Minor fixes" part. You should be more specific about which commands support `--ssh-public-key`

Thanks for your suggestion. I have updated the history.rst file.